### ISSUE TITLE
X axis MGN12

### ISSUE THREAD

**Issue by DiamonBlu** [Printopal/VoronDesign#1]
hi, do you have the file stl X axis MGN12 yet?

**Comment by Printopal** [comment]
Hi, thanks for asking. It's not available yet! I plan to modifiy the X carriage and install an Omron probe instead of the actual PL-08. The reason i am not sharing those files is i am getting overheating on that area and don't want to share anything until I solve this. Right now I don't have any time for this but I expect to get everything functional in august.
BTW if you already have an Omron probe, I think the X carriage files provided with the Trident should work and in that case I can share you the XY joints files.
Feel free to contact me on discord: Huttah#8854

**Comment by DiamonBlu** [comment]
oh yes, i will need the xy file, thanks so much 🤗

**Comment by Printopal** [comment]
Alright, I created an OBJ folder and uploaded the XY joints files.

Keep in mind they are designed for Gates Idler that are 10 mm tall.
You will need to add some few supports for the PCB clearance of xy_joint_right_lower.
You will also need to add some few supports for 1 M5 nut hole of xy_joint_right_lower, I decided to remove the chamfer of that M5 nut hole to have more thickness in Z.

Don't hesitate to give me your feedback :+1:
Happy printing!

**Comment by DiamonBlu** [comment]
yes, definitely, but i can't download the file, does it look like the file is corrupted or i didn't download it properly? can you check again?

**Comment by Printopal** [comment]
Edit:
More convenient way: I just added a compressed folder with the 4 obj files.

When you are in the OBJ folder, click on any file you want, then "view raw" to open the code line of the file. Right click and "save as" be sure to remove the .txt extension when you save the file as.

**Comment by DiamonBlu** [comment]
I downloaded it successfully, thank you for the tutorial. I will give you feedback when it is successfully installed
